pinecone/client_test.go

```diff
@@ -255,6 +255,14 @@ func (ts *IntegrationTests) TestCreatePodIndex() {
 	require.Equal(ts.T(), name, idx.Name, "Index name does not match")
 }
 
+func (ts *IntegrationTests) TestCreatePodIndexMissingReqdFields() {
```

Comment:
I think you could argue these are unit tests, since they aren't actually making network calls. I think you could update them to be unit tests instead, right?